examples/getting-started/README.md

```diff
@@ -3,83 +3,162 @@
 
 - A HubSpot account with CMS Free, Starter, Pro, or Enterprise access
 - Node.js v18.0 or higher and the associated version of NPM
 
-## Creating and Deploying a CMS React Project
+## Creating and deploying a CMS React project
+Before we dive into running your first CMS React project, its important to understand 3 significant directories that you will come across in all CMS React projects:
 
-### 1. Clone the Repo
+- `*-project`: The HubSpot project directory that gets uploaded via the Projects system to build and deploy your CMS React code found in the `/src` directory.
+- `*-theme`: This is where your HubL templates and theme configuration lives.
+- `*-project/*-app`: A package inside your project directory containing all your assets, styles, components, modules, islands, and any local development tooling you need, such as ESLint, Prettier, TypeScript, etc.
 
-First, clone the CMS React repository:
+### Clone the `getting-started` directory
+From your local directory for the following commands:
 
 ```bash
-git clone https://github.com/HubSpot/cms-react.git
+git clone --filter=blob:none --no-checkout https://github.com/HubSpot/cms-react.git;
+cd cms-react;
+git sparse-checkout set --cone;
+git checkout main;
+git sparse-checkout set examples/getting-started;
 ```
 
-CMS React projects have a 3 significant directories:
-
-- `getting-started-project`: The HubSpot project directory that gets uploaded via the Projects system to build and deploy your CMS React code found in the `/src` directory.
-- `getting-started-project/getting-started-app`: A package inside your project directory containing all your assets, styles, components, modules, islands, and any local development tooling you need, such as ESLint, Prettier, TypeScript, etc.
-- `getting-started-theme`: This is where your HubL templates and theme configuration lives.
-
-At the root of this example, you will configure any local dev tooling you want, such as ESLint, Prettier, TypeScript, etc.
-
-### 2. Install Dependencies
+This will clone only the `getting-started` directory from within cms-react repository down to your local file system.
 
-Navigate to the `cms-react/examples/getting-started` directory in your terminal and run:
-
-```bash
-npm install
-```
+### Setting up your `hubspot.config.yaml` file
+In order to develop locally and deploy your code to your HubSpot portal. You will need to configure a `hubspot.config.yaml` file. Run `hs init` from the root of the repository and follow the prompts to setup your `hubspot.config.yaml` file. Keep in mind that when setting up JS assets for the first time, you will need to deactivate and regenerate your personal access key, ensuring it includes `CMS Pages`, `Design Manager`, `Developer Projects`, and `GraphQL Data Fetching` permissions. See the [HubSpot CLI documentation](https://developers.hubspot.com/docs/cms/guides/getting-started-with-local-development) for more information.
```

Comment:
We don't want to imply that they need `GraphQL Data Fetching` permission to use CMS React, because that is only for enterprise.

Reply:
https://github.com/HubSpot/cms-react/pull/75/commits/b356a736697fd75f100c7b8779cde970535b8065